core/components/molecules/_action-input/_action-input.js

```diff
@@ -9,6 +9,14 @@ import { multiply } from '../../_helpers/pixel-calc'
 import { spacing } from '@auth0/cosmos-tokens'
 import { actionShape } from '@auth0/cosmos/_helpers/action-shape'
 
+const topMarginForActionsWithSize = size =>
+  ({
+    small: '0',
+    compressed: '2px',
+    large: '8px',
+    default: '6px'
+  }[size])
```

Comment:
Hey, minor question here: Is there any value on dynamically generating a new object on every function invocation?

Reply:
Since this is calculated for the Form Field component, shouldn't this be stored near that component?

Reply:
@cristiandouce Changing it now, there's no value on doing that. Thank you!
@landitus Yes, will move it to that one